app/queue.py

```diff
@@ -193,8 +219,9 @@ def acknowledge(self, receipt: UUID) -> bool:
         put_batch_saving_inflight_processed(self.__metrics_logger, self, 1)
         return True
 
-    def publish(self, message: str):
-        self._redis_client.rpush(self._inbox, message)
+    @sign_param
```

Comment:
This will sign the parameter automatically, using the `app.notify_signer` object.